src/ResourceManagement/Resource/Microsoft.Azure.Management.V2.Resource/Core/CreatableResources.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.V2.Resource.Core.CollectionActions;
+using Microsoft.Azure.Management.V2.Resource.Core.ResourceActions;
+using System.Threading;
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
please remove license header from line 7 and 8. You already have it at the top of the file